src/components/cylc/GraphSubgraph.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <g v-if="subgraph.label!='margin'" class="c-graph-subgraph">
+    <rect
+      :width="subgraph.width"
+      :height="subgraph.height"
+      :x="subgraph.x"
+      :y="subgraph.y"
+      rx="50"
+      ry="50"
+      fill="none" />
+    <text
+      :x="calcLabelXPosition"
+      :y="calcLabelYPosition"
```

Comment:
It's a computed property now so it should get a "thing" name rather than a "do" name to make it clear what it is:

```suggestion
      :x="labelXPosition"
      :y="labelYPosition"
```